### PR TITLE
Fix Postmark outbound webhook job

### LINKED ISSUE #2592: ErrorException: preg_match(): Unknown modifier '@'
Sentry Issue: [APIARY-3J](https://sentry.io/organizations/robojackets/issues/3329898494/?referrer=github_integration)

```
ErrorException: preg_match(): Unknown modifier '@'
  File "/app/Jobs/ProcessPostmarkOutboundWebhook.php", line 66, in Illuminate\Foundation\Bootstrap\HandleExceptions::handleError
    if (1 === preg_match('(?P<uid>[a-z]+[0-9]+)@gatech\.edu', $email, $matches)) {
  File "/app/Jobs/ProcessPostmarkOutboundWebhook.php", line 66, in App\Jobs\ProcessPostmarkOutboundWebhook::handle
    if (1 === preg_match('(?P<uid>[a-z]+[0-9]+)@gatech\.edu', $email, $matches)) {
  File "/artisan", line 37
    new Symfony\Component\Console\Output\ConsoleOutput
...
(34 additional frame(s) were not displayed)
```

### PR DESCRIPTION
**Pull request by kberzinch**
PHPstan was right again

Fixes #2592

### CODE REVIEW

File: phpstan.neon [hunk 1]
```diff
@@ -157,7 +157,6 @@ parameters:
     - '#Property App\\Models\\Payment::\$(?:amount|processing_fee) \(string(?:\|null)?\) does not accept (?:float|int|float\|int)\.#'
     - '#Property App\\Models\\Signature::\$user_agent \(string\|null\) does not accept array\|string\|null\.#'
     - '#Property Spatie\\Permission\\Contracts\\(Permission|Role)::\$name \(string\) does not accept mixed\.#'
-    - '#Regex pattern is invalid: Unknown modifier ''@'' in pattern: .+#'
```
Comment (evan10s) [inline phpstan.neon:160]: I should probably look more closely at what you're ignoring in the future

**Review by evan10s — APPROVED** [review]

File: phpstan.neon (hunk 1 above)
Comment (kberzinch) [reply phpstan.neon:160]: Well also phpstan is wrong a lot so if i don't already know it's correct I'll probably ignore it

File: app/Models/DuesPackage.php [hunk 2]
```diff
@@ -235,8 +235,8 @@ public function scopeActive(Builder $query): Builder
      */
     public function scopeAccessActive(Builder $query): Builder
     {
-        return $query->where('access_start', '<', date('Y-m-d H:i:s'))
-            ->where('access_end', '>', date('Y-m-d H:i:s'));
+        return $query->where('access_start', '<', now())
+            ->where('access_end', '>', now());
```
Comment (JoshOldenburg) [inline app/Models/DuesPackage.php:239]: Why these changes?

Comment (kberzinch) [reply app/Models/DuesPackage.php:239]: It's just cleaner.

In other places where the timestamp portion was previously omitted, there is an actual functional change: `now()` is more precise.
